src/test/regress/sql/multi_mx_call.sql

```diff
@@ -0,0 +1,30 @@
+-- Test passing off CALL to mx workers
```

Comment:
Maybe extend the tests to:
1. Check for cases that we shouldn't route? Set `client_min_messages`, and maybe create a case for each of the `return false`s in `CallFuncExprRemotely`.
2. Can procedures return set of records, or other kinds of sets? Maybe check for that.
3. Add a real MX table test.
4. Maybe add some cache related tests. For example, verify that UPDATE/DELETEs invalidate cache correctly.
...